source/ShopPanel.h

```diff
@@ -83,7 +83,7 @@ class ShopPanel : public Panel {
 	// These are for the individual shop panels to override.
 	virtual int TileSize() const = 0;
 	virtual int VisibilityCheckboxesSize() const;
-	virtual int DrawPlayerShipInfo(const Point &point) = 0;
+	virtual int DrawPlayerShipInfo(const Point &point);
```

Comment:
Does it still need to be virtual?

Reply:
Probably not?  Someone might want to make changes to one or the other in the future?

Reply:
It shouldn't be virtual anymore. The reason it was virtual before was so that it could be called from within members of ShopPanel but the implementation from the appropriate child node would be used, and so it doesn't need to be defined in ShopPanel.

It should probably also be made private now because it doesn't look to me like it is referred to by the children.

Reply:
Well, fair enough.  I noticed a couple of other functions that also didn't get called by the kids, so I moved those too, as they were all in a group.